packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1082,9 +1088,9 @@ export default class Meetings extends WebexPlugin {
       const activeLocusUrl = [];
 
       if (locusArray?.loci && locusArray.loci.length > 0) {
-        locusArray.loci.forEach((locus) => {
+        locusArray.loci.forEach(async (locus) => {
```

Comment:
`forEach` does not wait for each of the functions invoked on each array element to resolve
see https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

Reply:
Makes sense @marcin-bazyl 

Refactored it to the `for..of` loop. Thank you.